serverless/puppeteer-screenshots/serverless.yml

```diff
@@ -0,0 +1,40 @@
+service: puppeteer-screenshots
+
+provider:
+  name: aws
+  runtime: nodejs6.10
+  memorySize: 1536
+  timeout: 180
+  iamRoleStatements:
+   -  Effect: "Allow"
+      Action:
+        - "s3:GetObject"
+        - "s3:PutObject"
+        - "s3:PutObjectAcl"
+      Resource: "arn:aws:s3:::*"
+   -  Effect: Allow
+      Action:
+        - sns:Publish
+      Resource: "arn:aws:sns:us-east-1:802922962628:*"
```

Comment:
We can use serverless env vars to replace all these references to the ID so it can move with our account and it's not publicly shared.